cdap-proto/src/test/java/co/cask/cdap/proto/id/EntityIdTest.java

```diff
@@ -269,4 +270,7 @@ public void printToJson() {
     }
   }
 
+  private Map<String, Object> jsonToMap(String json) {
+    return GSON.fromJson(json, new TypeToken<Map<String, Object>>() { }.getType());
```

Comment:
put `new TypeToken<Map<String, Object>>() { }.getType()` into a static final variable. No need to create a new instance each time.